config/imagery/gebco_2020_Nztm2000Quad_305-75m-NZTM2000Quad.json

```diff
@@ -20,5 +20,6 @@
       "height": 10018754.171400001,
       "name": "0-0-0"
     }
-  ]
+  ],
+  "title": "Gebco 2020.nc"
```

Comment:
GEBCO Gridded Bathymetry (2020)

I'm not exactly sure what the source data is for this and what it should be called, is it https://www.gebco.net/data_and_products/gridded_bathymetry_data/gebco_2020/ ?